### PR TITLE
Isolated install

### PR DESCRIPTION
**Pull request by julianbermudez**
This pull request changes the `UCLIENT_MERGE_INSTALL` flag by `UCLIENT_ISOLATED_INSTALL`, simplifies the isolated installation policy and adds tests for isolated and non-isolated install.

- [x] Please, once approved https://github.com/eProsima/Micro-CDR/pull/48, change `_microcdr_tag` to develop branch.

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Linux/72/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Windows/73/)

File: ci/linux/CMakeLists.txt [hunk 1]
```diff
@@ -61,5 +61,35 @@ ExternalProject_Add(microxrcedds_client
         -DCMAKE_SHARED_LINKER_FLAGS:STRING=${_shared_linker_flags}
         -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>
         -DUCLIENT_BUILD_CI_TESTS:BOOL=ON
+        -DUCLIENT_MERGE_INSTALL:BOOL=OFF
```
Comment (BorjaOuterelo) [inline ci/linux/CMakeLists.txt]: UCLIENT_ISOLATED_INSTALL?

File: ci/linux/CMakeLists.txt [hunk 2]
```diff
@@ -61,5 +61,35 @@ ExternalProject_Add(microxrcedds_client
...
+        -DCMAKE_CXX_COMPILER:FILEPATH=${CMAKE_CXX_COMPILER}
+        -DCMAKE_C_COMPILER:FILEPATH=${CMAKE_C_COMPILER}
+        -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}
+        -DCMAKE_C_FLAGS:STRING=${_c_flags}
+        -DCMAKE_CXX_FLAGS:STRING=${_cxx_flags}
+        -DCMAKE_EXE_LINKER_FLAGS:STRING=${_exe_linker_flags}
+        -DCMAKE_SHARED_LINKER_FLAGS:STRING=${_shared_linker_flags}
+        -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>
+        -DUCLIENT_BUILD_CI_TESTS:BOOL=ON
+        -DUCLIENT_MERGE_INSTALL:BOOL=ON
```
Comment (BorjaOuterelo) [inline ci/linux/CMakeLists.txt]: UCLIENT_MERGE_INSTALL?

File: test/case/CMakeLists.txt [hunk 3]
```diff
@@ -30,7 +29,8 @@ add_test(
     COMMAND
         ${CMAKE_COMMAND}
             -DORIGINAL_DIR=${CMAKE_CURRENT_SOURCE_DIR}/packaging
-            -DCMAKE_PREFIX_PATH=${CMAKE_INSTALL_PREFIX}
+            -DCMAKE_PREFIX_PATH=${_prefix_path}
```
Comment (BorjaOuterelo) [inline test/case/CMakeLists.txt]: ${_prefix_path} Is still valid?

File: ci/linux/CMakeLists.txt (hunk 1 above)
Comment (julianbermudez) [reply ci/linux/CMakeLists.txt]: Fixed!

File: ci/linux/CMakeLists.txt (hunk 2 above)
Comment (julianbermudez) [reply ci/linux/CMakeLists.txt]: Fixed!

File: test/case/CMakeLists.txt (hunk 3 above)
Comment (julianbermudez) [reply test/case/CMakeLists.txt]: Fixed!

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Linux/73/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Windows/74/)

File: CMakeLists.txt [hunk 4]
```diff
@@ -48,7 +48,7 @@ set(UCLIENT_CONFIG "${CMAKE_CURRENT_SOURCE_DIR}/client.config" CACHE PATH "Confi
 # Dependencies
 ###############################################################################
 set(_microcdr_version 1.1.2)
-set(_microcdr_tag v1.1.2)
+set(_microcdr_tag feature/prefix)
```
Comment (BorjaOuterelo) [inline CMakeLists.txt]:
```suggestion
set(_microcdr_tag develop)
```

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Linux/75/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Client%20Manual%20Windows/76/)

**Pull request merged**